README.md

```diff
@@ -107,23 +107,12 @@ Con un semplice Click permette di Passare da un profilo di Firefox all'altro!
 
 ### Altri progetti Mozilla
 
-Sono numerosi i progetti portati avanti da Mozilla, ma sicuramente quelli elencati qui sotto sono i più importanti e conosciuti. Tra quelli che non ho citato ci sono anche Firefox for Android, Mozilla Sync, il progetto WebMaker, Mozilla Science, Mozilla-Knights OpenNews, Mozilla Advocacy; come potrai notare facendo qualche ricerca, si rivolgono a persone con molte competenze diverse (insegnanti, scienziati, giornalisti).
+Numerosi sono i progetti portati avanti dalla fondazione Mozilla, e tra quelli non elencati ci sono progetti del calibro di Mozilla Sync, WebMaker, Mozilla Science, Mozilla-Knights OpenNews, Mozilla Advocacy, Mozilla Winter of Security. Tutti questi progetti si rivolgono a persone con differenti competenze (ad esempio: Insegnanti, Scienziati, Giornalisti e Studenti) e sono legati da un unico comune denominatore: la visione del web come risorsa libera, aperta a tutte le popolazioni del mondo. In Dettaglio tra i progetti di Mozilla troviamo il Programma di Posta Thunderbird.
```

Comment:
Io preferisco fare la revisione su github che é pubblica, nei progetti open si fa cosi.
Il fatto che c'erano anche prima non significa che vanno bene lo stesso XD
Per winter of security hai ragione, però lo ritengo un progetto troppo specifico e poco conosciuto mentre promuovere progetti piŭ di portata piŭ ampia come rust o mdn é meglio.
Inoltre wos é solo per persone preparate che hanno un certo background non é per meri utilizzatori.